CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
 ﻿cmake_minimum_required(VERSION 3.18) # needed on win32
 
-project("purple-gowhatsapp" LANGUAGES C)
+set(PROJECT_NAME "purple-gowhatsapp" C)

```

Comment:
What is the purpose of defining the project name this way? It looks like the the "LANGUAGES" option is dropped and building fails on systems without a C++ compiler.

Reply:
An oversight if this breaks or removes functionality, I was simply following the skype4pidging packaging.  I didn't intend to modify the build!

I suspect we can keep the original code -- note the later reference to PROJECT_NAME will be removed, and since it's never used again -- it only appears after being set and then used in CMAKE_PROJECT_NAME for packages and file names later.

Thoughts ?

Reply:
I reverted to the simple `project(…)` command. It looks like it is working on all of my machines (Debian, arch and win32).